src/plugins/analysis/input_vectors/internal/input_vectors_r2.py

```diff
@@ -0,0 +1,122 @@
+import base64
+import json
+import os
+import re
+import sys
+
+import r2pipe
+
+
+def open_with_radare(path_to_elf):
+    r2 = r2pipe.open(path_to_elf)
+    r2.cmd('aaaa')
+    return r2
+
+
+def get_ins(r2, func):
```

Comment:
if `r2` gets handed to almost every function, this is a sign that using a class could make sense

Reply:
Future work :)